### PR TITLE
Security Profiles Operator 0.8.2 release notes

### PR DESCRIPTION
**Pull request by sheriff-rh**
Version(s):
4.12+

Issue:
None 

Link to docs preview:
[Security Profiles Operator 0.8.2](https://file.rdu.redhat.com/antaylor/SPO-0.8.1/security/security_profiles_operator/spo-release-notes.html#spo-release-notes-0-8-2)

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by sheriff-rh** [comment]
@xiaojiey @Vincent056  please review when you get a chance, thanks.

**Comment by sheriff-rh** [comment]
@Vincent056  @xiaojiey @BhargaviGudi  please review as soon as you can so that I can have this peer reviewed by my team as well prior to release. Thank you!

**Review by Vincent056 — APPROVED** [review]
/lgtm
Sorry for the late review on this

File: security/security_profiles_operator/spo-release-notes.adoc [hunk 1]
```diff
@@ -12,6 +12,24 @@ These release notes track the development of the Security Profiles Operator in {
...
+== Security Profiles Operator 0.8.1
+
+The following advisory is available for the Security Profiles Operator 0.8.1:
+
+* link:https://access.redhat.com/errata/RHBA-2023:5958[RHBA-2023:5958 - OpenShift Security Profiles Operator bug fix update]
+
+[id="spo-0-8-1-bug-fixes"]
+=== Bug fixes
+
+* Previously, SELinuxProfiles did not inherit custom SELinuxProfiles from the same namespace. With this update, the issue has now been resolved and SELinuxProfiles are inherited from the same namespace as expected. (link:https://issues.redhat.com/browse/OCPBUGS-17164[*OCPBUGS-17164*])
```
Comment (GroceryBoyJr) [inline security/security_profiles_operator/spo-release-notes.adoc:25]:
```suggestion
* Previously, `SELinuxProfiles` did not inherit custom SELinuxProfiles from the same namespace. With this update, the issue has now been resolved and SELinuxProfiles are inherited from the same namespace as expected. (link:https://issues.redhat.com/browse/OCPBUGS-17164[*OCPBUGS-17164*])
```

**Review by GroceryBoyJr — COMMENTED** [review]
Only one nit, @sheriff-rh . Otherwise, LGTM.

**Comment by sheriff-rh** [comment]
@xiaojiey please review this PR as soon as possible. Thanks!

File: security/security_profiles_operator/spo-release-notes.adoc [hunk 2]
```diff
@@ -12,6 +12,24 @@ These release notes track the development of the Security Profiles Operator in {
...
+=== Bug fixes
+
+* Previously, `SELinuxProfiles` did not inherit custom `SELinuxProfiles` from the same namespace. With this update, the issue has now been resolved and `SELinuxProfiles` are inherited from the same namespace as expected. (link:https://issues.redhat.com/browse/OCPBUGS-17164[*OCPBUGS-17164*])
+
+* Previously, RawSELinuxProfiles would hang during the creation process and would not reach an `Installed` state. With this update, the issue has been resolved and RawSELinuxProfiles are created successfully. (link:https://issues.redhat.com/browse/OCPBUGS-19744[*OCPBUGS-19744*])
+
+* Previously, patching the `enableLogEnricher` to `true` would cause the `seccompProfile` `log-enricher-trace` pods to be stuck in a `Pending` state. With this update, `log-enricher-trace` pods reach an `Installed` state as expected. (link:https://issues.redhat.com/browse/OCPBUGS-22182[*OCPBUGS-22182*])
+
+* Previously, the Security Profiles Operator generated high cardinality metrics, causing Prometheus pods using high amounts of memory. With this update, cardinality is reduced and the Prometheus pods use less memory. (link:https://issues.redhat.com/browse/OCPBUGS-22406[*OCPBUGS-22406*])
```
Comment (xiaojiey) [inline security/security_profiles_operator/spo-release-notes.adoc:31]: Not sure "cardinality is reduced" is accurate or not. With this date, the below 3 metrics will disappear for the SPO namespace.
rest_client_request_duration_seconds
rest_client_request_size_bytes
rest_client_response_size_bytes
Screenshot from 2023-12-14 14-21-58

File: security/security_profiles_operator/spo-release-notes.adoc [hunk 3]
```diff
@@ -12,6 +12,24 @@ These release notes track the development of the Security Profiles Operator in {
...
+== Security Profiles Operator 0.8.1
+
+The following advisory is available for the Security Profiles Operator 0.8.1:
+
+* link:https://access.redhat.com/errata/RHBA-2023:5958[RHBA-2023:5958 - OpenShift Security Profiles Operator bug fix update]
+
+[id="spo-0-8-1-bug-fixes"]
+=== Bug fixes
+
+* Previously, `SELinuxProfiles` did not inherit custom `SELinuxProfiles` from the same namespace. With this update, the issue has now been resolved and `SELinuxProfiles` are inherited from the same namespace as expected. (link:https://issues.redhat.com/browse/OCPBUGS-17164[*OCPBUGS-17164*])
```
Comment (xiaojiey) [inline security/security_profiles_operator/spo-release-notes.adoc:25]: Could you please help to update all `SELinuxProfiles` to `SelinuxProfile`, or least the second one? It is case sentitive for the yaml file.
```
$ oc apply -f -<<EOF
> apiVersion: security-profiles-operator.x-k8s.io/v1alpha2
kind: SelinuxProfile
metadata:
  name: test-inherit-policy-selinuxprofile
  namespace: nginx-deploy
spec:
  inherit: 
  - kind: SELinuxProfiles
    name: nginx-secure
  allow:
    var_run_t:
      sock_file:
        - write
EOF
The SelinuxProfile "test-inherit-policy-selinuxprofile" is invalid: spec.inherit[0].kind: Unsupported value: "SELinuxProfiles": supported values: "System", "SelinuxProfile"
```

Comment (sheriff-rh) [reply security/security_profiles_operator/spo-release-notes.adoc:25]: Yes - I can change this to `SelinuxProfile` objects. :+1:

File: security/security_profiles_operator/spo-release-notes.adoc (hunk 2 above)
Comment (sheriff-rh) [reply security/security_profiles_operator/spo-release-notes.adoc]: Done: 
image

**Pull request merged**